<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SGTView</class>
 <widget class="QWidget" name="SGTView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1820</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="GroupBox" name="gpb_SGT">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>45</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="spacing">
         <number>10</number>
        </property>
        <item row="9" column="3">
         <widget class="QSpinBox" name="spb_Buzzer_Off">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QSpinBox" name="spb_BlueLight_On">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Red Light</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLabel" name="lbl_SGTState">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QComboBox" name="cmb_BuzzerTwinkleMode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QSpinBox" name="spb_YellowLight_Off">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QSpinBox" name="spb_YellowLight_On">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Yellow Light</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QSpinBox" name="spb_Buzzer_On">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="spb_RedLight_On">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="cmb_YellowLightTwinkleMode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>OFF</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="Line" name="line_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>3</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:3px solid qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgb(205, 235, 255), stop:1 rgb(210, 180, 225));</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Green Light</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QSpinBox" name="spb_RedLight_Off">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_7">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Buzzer</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QSpinBox" name="spb_GreenLight_On">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_8">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Model</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QSpinBox" name="spb_BlueLight_Off">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QSpinBox" name="spb_GreenLight_Off">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ON</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="cmb_BlueLightTwinkleMode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="10" column="2" colspan="2">
         <widget class="QPushButton" name="btn_Confirm">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Current State</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Blue Light</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="cmb_RedLightTwinkleMode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="cmb_GreenLightTwinkleMode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Status</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QComboBox" name="cmb_SGTStatus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_MaunalTest">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_MaunalTestExit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1322</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>CustomerWidget/GroupBox/groupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
